.github/workflows/deploy-examples.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy example apps to Fly
+
+on:
+  # Allow calling this workflow from other workflows
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        description: "Wasp version to use for deployment (default to latest)"
+        required: false
+    secrets:
+      FLY_API_TOKEN:
+        description: "Fly API token for deploying apps"
+        required: true
+
+  # Allow calling this workflow manually from the GitHub UI
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Wasp version to use for deployment (default to latest)"
+        required: false
+
+  # TODO: REMOVE THIS BEFORE MERGING, JUST FOR TESTING
+  push:
+    branches:
+      - 2831-automatically-deploy-all-example-apps-on-release
+
+concurrency:
+  group: "deploy-examples"
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    strategy:
+      matrix:
+        projects:
```

Comment:
`thoughts` is also present in the `release` branch, but it has been removed from `main` and not deployed anymore

Reply:
What about `waspello`?

Reply:
> I think we should, yes!

On it

> I'm wondering whether we should also deploy the tutorials? Wdyt?
 
No strong opinion, but leaning towards no. Examples are deployed so people can play with them without doing it yourself. The value in the tutorials is that people _do_ do them themselves. And the end state isn't something really worth showcasing.